chips/lpc55/chip.toml

```diff
@@ -46,6 +46,12 @@ size = 4096
 address = 0x4003A000
 size = 4096
 
+# this is the start of the USB SRAM AHB peripheral (0x4000 bytes total)
+# we appropriate this SRAM for passing DICE artifacts
+[dice_alias]
```

Comment:
Normally I'd agree with you here but eventually we'll be transferring other collections of artifacts across this boundary. In a future where we're transferring credentials to multiple recipients / hubris tasks we'll want guarantees that each task is only able to access the appropriate credentials. AFAIK, by dividing the USB SRAM into regions in the chip.toml we're able to grant access to each region using the existing mechanisms in the kernel. I wrote this up in RFD 282: https://rfd.shared.oxide.computer/rfd/0282#_ram

Use of the USB SRAM was a choice of convenience and so if the issue is with this memory being associated with a peripheral we could select any sufficiently large & unused region of memory for this purpose. This would still require that we list it in the chip.toml file as separate memory regions but it would keep us from labeling the USB SRAM as anything but what it is.

Reply:
Could we use SRAM 4 instead (`0x20040000-0x20043FFF`)?  That seems less weird than using spare USB SRAM, and isn't otherwise used in the system.

(I'm not sure if we'd need to jump through any hoops at startup to turn it on, or whether it's on immediately)